moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -166,6 +166,9 @@ class IKFastKinematicsPlugin : public kinematics::KinematicsBase
   const std::string IKFAST_TIP_FRAME_ = "_EEF_LINK_";
   const std::string IKFAST_BASE_FRAME_ = "_BASE_LINK_";
 
+  // prefix added to tip- and baseframe to allow different namespaces or multi-robot setups
+  std::string linkframeprefix_;
```

Comment:
I would prefer if this member name was [under_scored](http://wiki.ros.org/CppStyleGuide#Naming)